integrations-and-supported-tools/tensorboard/scripts/requirements.txt

```diff
@@ -0,0 +1,3 @@
+neptune
+neptune-tensorboard==1.0.0rc2
```

Comment:
Will update to `neptune[tensorboard]` once https://github.com/neptune-ai/neptune-client/pull/1368 is in

Reply:
thanks!
will merge this now. Need to update the installation to `neptune[tensorboard]` once the client is updated